### PR TITLE
fix: [ANDROAPP-3374] Event list should have rounded top corners

### PR DESCRIPTION
**Pull request by Balcan**
## Description
Please include a summary of the change and include the related jira issue if it exists.

[ jira issue ](https://jira.dhis2.org/browse/ANDROAPP-3374)

## Solution description
If this PR is a fix include a brief description on how the issue is solved.
## Covered unit test cases
Describe the tests that you ran to verify your changes.
## Where did you test this issue?
- [ ] Smartphone Emulator
- [ ] Tablet Emulator
- [ ] Smartphone
- [ ] Tablet
## Which Android versions did you test this issue?
- [ ] 4.4
- [ ] 5.X - 6.X
- [ ] 7.X
- [ ] 8.X
- [ ] 9.X - 10.X
- [ ] Other
## Checklist
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code

### CODE REVIEW

File: app/src/main/java/org/dhis2/Bindings/ViewExtensions.kt [hunk 1]
```diff
@@ -38,3 +41,20 @@ fun TextView.clearFocusOnDone() {
         }
     }
 }
+
+fun View.clipWithRoundedCorners(curvedRadio: Int = 16.dp) {
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+        outlineProvider = object : ViewOutlineProvider() {
+            override fun getOutline(view: View, outline: Outline) {
```
Comment (ferdyrod) [inline app/src/main/java/org/dhis2/Bindings/ViewExtensions.kt:48]: 🔝 Didn't knew about this. nice!!!

File: app/src/main/java/org/dhis2/usescases/programEventDetail/ProgramEventDetailActivity.java [hunk 2]
```diff
@@ -14,6 +16,7 @@
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
+import android.view.ViewOutlineProvider;
```
Comment (ferdyrod) [inline app/src/main/java/org/dhis2/usescases/programEventDetail/ProgramEventDetailActivity.java:19]: All these imported classes (Outline, Build, ViewOutlineProvider) are not needed in here.

File: app/src/main/java/org/dhis2/Bindings/ViewExtensions.kt (hunk 1 above)
Comment (Balcan) [reply app/src/main/java/org/dhis2/Bindings/ViewExtensions.kt:48]: https://youtu.be/0p_1QSUsbsM?t=10

Comment (ferdyrod) [reply app/src/main/java/org/dhis2/Bindings/ViewExtensions.kt:48]: And here I thought it was a video talking about the ViewOutlineProvider!  🤣 🤣 🤣

File: app/src/main/java/org/dhis2/Bindings/ViewExtensions.kt [hunk 3]
```diff
@@ -38,3 +41,20 @@ fun TextView.clearFocusOnDone() {
         }
     }
 }
+
+fun View.clipWithRoundedCorners(curvedRadio: Int = 16.dp) {
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+        outlineProvider = object : ViewOutlineProvider() {
```
Comment (JaimeToca) [inline app/src/main/java/org/dhis2/Bindings/ViewExtensions.kt:47]: can we reouse this clip rounded in home screen?

Comment (Balcan) [reply app/src/main/java/org/dhis2/Bindings/ViewExtensions.kt:47]: We can use it everywhere and dismiss the background. But i would create a separete issue for that

File: app/src/main/java/org/dhis2/usescases/programEventDetail/ProgramEventDetailActivity.java (hunk 2 above)
Comment (Balcan) [reply app/src/main/java/org/dhis2/usescases/programEventDetail/ProgramEventDetailActivity.java]: done!

**Review by ferdyrod — APPROVED** [review]

File: app/src/main/java/org/dhis2/Bindings/ViewExtensions.kt (hunk 3 above)
Comment (Balcan) [reply app/src/main/java/org/dhis2/Bindings/ViewExtensions.kt:47]: Adding it in ANDROAPP-3372 as it will help to other bug

**Review by JaimeToca — APPROVED** [review]

**Pull request merged**